packages/@ionic/cli/src/lib/integrations/capacitor/index.ts

```diff
@@ -190,7 +191,16 @@ export class Integration extends BaseIntegration<ProjectIntegration> {
 
     debug('Getting config with Capacitor CLI: %O', args);
 
-    const output = await this.e.shell.cmdinfo('capacitor', args, { cwd: this.root });
+    const output = await ((cmd: string, opts: ExecOptions): Promise<string | undefined> => {
+      return new Promise((resolve, reject) => {
+        exec(cmd, opts, (error, stdout, stderr) => {
+          if (error) {
+            resolve(undefined);
```

Comment:
Would we want to reject the promise here?

Reply:
Yep, good catch, this was left over from development testing 😅